packages/core/package.json

```diff
@@ -32,9 +36,6 @@
   },
   "homepage": "https://github.com/eslint/rewrite#readme",
   "devDependencies": {
-    "@types/eslint": "^9.6.0",
-    "eslint": "^9.0.0",
-    "mocha": "^10.4.0",
     "typescript": "^5.4.5"
```

Comment:
It looks like only the `typescript` dependency is used during the build (to run the `tsc` command). `eslint` is only used when linting from the root of the monorepo, where it is already included in the `devDependencies`.